### PR TITLE
Use the repo URL to avoid requiring to create a release

### PR DESCRIPTION
**Pull request by randradas**
**Description of the change**

Better use the repo URL instead of a release one.

### CODE REVIEW

File: chart/tests/carto.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "package": {
       "context": {
         "resources": {
-          "url": "https://github.com/CartoDB/carto3-helm/archive/refs/tags/v0.0.4.tar.gz",
```
Comment (paleloser) [inline chart/tests/carto.json:6]: I'd use a commit instead, so we make sure the pipeline is reproducible and that it should work: `https://github.com/CartoDB/carto3-helm/tarball/c321361a7a237731c05572876d147cdba5190569`. WDYT?

Comment (randradas) [reply chart/tests/carto.json:6]: I agree but this commit is older than the PR itself, does it make sense?

Comment (paleloser) [reply chart/tests/carto.json:6]: I guess it depends of the goal of this pipeline: if it is to stay up-to-date to the code, regardless if it succeeds or not then it doesn't. But if this pipeline aims to illustrate how the engine works, with a working pipeline, then I'd fix it to a commit where we know it worked.

Comment (randradas) [reply chart/tests/carto.json:6]: it's just to illustrate, done!

File: chart/tests/carto.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (paleloser) [inline chart/tests/carto.yaml:1]: `.json` right?

**Pull request merged**